workflow-templates/im-build-db-ci.yml

```diff
@@ -133,21 +136,214 @@ jobs:
         with:
           version: 7.2.0 # This version works with the current version of build-database-ci-action. Newer versions might, but they should be tested.
 
-      - name: Build Database
-        uses: im-open/build-database-ci-action@v2.0.1
+      # TODO: This can be removed if your database doesn't have database object dependencies that it needs to download and install in order to build and run tests.
+      - name: Install and run dependency scripts
+        uses: im-open/install-and-run-db-dependency-scripts@v1.1.0
         with:
-          db-server-name: ${{ env.DB_SERVER_NAME }}
-          db-server-port: ${{ env.DB_SERVER_PORT }}
-          db-name: ${{ env.DB_NAME }}
-          install-mock-db-objects: true # TODO: Remove this if you don't have any dependencies on objects in other repositories (the default value is false)
-          mock-db-object-nuget-feed-url: ${{ env.ARTIFACTORY_RETRIEVAL_URL }} # TODO: If install-mock-db-objects is set to false, this can be removed
-          drop-db-after-build: false
-          run-tests: true # TODO: Remove this if your database doesn't have tests.
-          nuget-username: ${{ secrets.ARTIFACTORY_USERNAME }} # TODO: If install-mock-db-objects is set to false, this can be removed
-          nuget-password: ${{ secrets.ARTIFACTORY_API_KEY }} # TODO: If install-mock-db-objects is set to false, this can be removed
-          # If you need to run this job on Windows runners, then remove db-username and db-password so integrated security will be utilized
-          db-username: sa
-          db-password: ${{ secrets.ACTIONS_RUNNER_MSSQL_SA_PWD }}
+          db-server-name: '${{ env.DB_SERVER_NAME }},${{ env.DB_SERVER_PORT }}'
+          db-name: '${{ env.DB_NAME }}'
+          username: '${{ env.SQL_LOGIN_USERNAME }}'
+          password: ${{ secrets.ACTIONS_RUNNER_MSSQL_SA_PWD }} # This is an org-level secret
+          # TODO: Fill in this list with real values.
+          dependency-list: >-
+            [
+            {
+            'version': '1.0.0',
+            'packageName': 'Some_Dependency',
+            'nugetUrl': '${{ env.ARTIFACTORY_URL }}/Path/To/Some_Dependency.1.0.0.nupkg'
+            },
+            {
+            'version': '1.0.0',
+            'packageName': 'Some_Other_Dependency',
+            'nugetUrl': '${{ env.ARTIFACTORY_URL }}/Path/To/Some_Other_Dependency.1.0.0.nupkg'
+            }
+            ]
```

Comment:
Can we add some indentation here so it is easier to read?  Will the indentation keep it from working right?